homeassistant/components/light/device_automation.py

```diff
@@ -20,29 +21,56 @@
 CONF_TURN_OFF = "turn_off"
 CONF_TURN_ON = "turn_on"
 
+ENTITY_CONDITIONS = [
+    {
+        # True when light is turned off
+        CONF_CONDITION: "device",
+        CONF_DOMAIN: DOMAIN,
+        CONF_TYPE: CONF_TURN_OFF,
+    },
+    {
+        # True when light is turned on
+        CONF_CONDITION: "device",
+        CONF_DOMAIN: DOMAIN,
+        CONF_TYPE: CONF_TURN_ON,
+    },
+]
+
 ENTITY_TRIGGERS = [
     {
-        # Trigger when light is turned on
+        # Trigger when light is turned off
         CONF_PLATFORM: "device",
         CONF_DOMAIN: DOMAIN,
         CONF_TYPE: CONF_TURN_OFF,
     },
     {
-        # Trigger when light is turned off
+        # Trigger when light is turned on
         CONF_PLATFORM: "device",
         CONF_DOMAIN: DOMAIN,
         CONF_TYPE: CONF_TURN_ON,
     },
 ]
 
+CONDITION_SCHEMA = vol.All(
+    vol.Schema(
+        {
+            vol.Required(CONF_CONDITION): "device",
+            vol.Optional(CONF_DEVICE_ID): str,
+            vol.Required(CONF_DOMAIN): DOMAIN,
+            vol.Required(CONF_ENTITY_ID): cv.entity_id,
+            vol.Required(CONF_TYPE): vol.In([CONF_TURN_OFF, CONF_TURN_ON]),
```

Comment:
this feels a bit weird, shouldn't this be `is_on` and `is_off` ? 

I also wonder if we should move some generic constants to `device_automation/const.py`

Reply:
Fixed.

Reply:
You say fixed but then we should update these constants to be `CONF_IS_ON` `CONF_IS_OFF` ?

Reply:
```suggestion
            vol.Required(CONF_TYPE): vol.In([CONF_IS_OFF, CONF_IS_ON]),
```

Reply:
Oops, now properly fixed!